inventory/byo/hosts.origin.example

```diff
@@ -113,6 +113,10 @@ openshift_release=v3.6
 # Downgrades are not supported and will error out. Be careful when upgrading docker from < 1.10 to > 1.10.
 # docker_version="1.12.1"
 
+# Specify whether to run Docker daemon with SELinux enabled in containers.
+# (Needs to be false to use overlay/overlay2 storage drivers before RHEL 7.4)
```

Comment:
nit: the True/False here is case-insensitive, isn't it? I'd suggest dropping the parentheses and make it a "command" statement:

```
Set to False when using the overlay or overlay2 storage driver on RHEL 7.3 or earlier.
```

What about other operating systems? CentOS, Fedora?

Is it clear that the commented-out values are the defaults? I'm not sure one can reason that way in the example inventory.